definitions/ext-ups/definition.yml

```diff
@@ -0,0 +1,27 @@
+domain: EXT
+type: UPS
+synthesis:
+  name: device_name
+  identifier: device_name
+  encodeIdentifierInGUID: true
+
+  conditions:
+  - attribute: provider
+    value: kentik-ups
+
+  tags:
+    src_addr:
+      entityTagName: device_ip
+
+compositeMetrics:
+  goldenMetrics:
+  - golden_metrics.yml
+  summaryMetrics:
+  - summary_metrics.yml
+
+goldenTags:
+- device_ip
+
+dashboardTemplates:
+  newRelic:
```

Comment:
Same as with the other PR, I think that you could use `kentik` here instead (that's the value of the `instrumentation.provider` for this telemetry right?) that way it'd be easier for people to extend this definition for other providers, what do you think?

Reply:
[same as the other PR] that's a good callout - I was thinking to wait until the new standard that @srvaroa and folks are working on is in place for using `instrumentation.provider` ++ ??? and then I could push an update to all of the Kentik entities at once. 

Would you rather I submit the change to `kentik` on this PR and then update later?

Reply:
per conversation on PR #236 - updating to `kentik` to match now and we will address later once new standard is in place